project-koku/koku
# [COST-3527] OCP unique label capture

## Jira Ticket

[COST-3527](https://issues.redhat.com/browse/COST-3527)

## Description

This change will populate new labels in the OCPEnabledTagKey table during report processing, defaulting to disabled.

## Testing

1. Checkout Branch
2. Run make create-test-customer and make load-test-customer-data
3. WAIT
4. Run SQL `select * from org1234567.reporting_ocpenabledtagkeys;` 
5. See all label keys in this table, defaulting to enabled=false except for those we expllicity enable.

## Notes

...
